portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -36,19 +37,30 @@
 
   private List<String> messages;
 
+  public static LinkedHashMap<String, Integer> commentsByPerson = new LinkedHashMap<>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Comment"); //.addSort("timestamp", SortDirection.DESCENDING);
+    Query query = new Query("Comment");
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
-    List<String> comments = new ArrayList<>();
+    List<String> comments = new ArrayList<String>();
```

Comment:
I like the method you used here for retrieving the user's data and displaying it with inner HTML, instead of using a fetch request. It's very simple and effective and I think it does save the programmer some time understanding how the information is being parsed here. In JS I find it harder to retrieve the information from the object returned in a response. Good Job!